cli/http.go

```diff
@@ -0,0 +1,115 @@
+package cli
```

Comment:
Does this still need to be in the CLI package? I did this originally as kind of a hack because I needed to call those methods.